_pages/dev/lazy-loading-guide.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Lazy Loading Guide
+---
+
+**Lazy loading or code splitting** lets you divide the JavaScript code into multiple chunks, so they don't have to be loaded upfront, but only when actually needed, greatly improving Time To Interactive, especially in case of complex web applications and on lower-end mobile devices. 
+
+## Spartacus Approach to Lazy Loading
+   
+As Spartacus is mostly CMS driven, there is not much of the usage of the built-in route based lazy loading offered by Angular out of the box. Code splitting has to be done at app build time, while Spartacus never knows upfront what components or features a specific route can contain, before actually loading them. 
+There is always a room for manual tweaking it based on app knowledge, but such a solution breaks the flexibility that Spartacus offers and requires additional prerequisites.
```

Comment:
While I understand what you're saying, and understand you like to be thorough. But perhaps we isolate this somewhere else. I can imagine we have a topic that covers exceptions/custom cases.